intro-to-tdd-with-java/scenario-student-tracking-application/assets/java/student-tracking-application/.idea/compiler.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I dont think the .idea directory is needed, can you remove it.

Reply:
good point. going to abandon this PR in favour of the TDD module Craig is about to submit ... but have passed this note to him.